src/components/DataTable/DataTableCell/DataTableCell.tsx

```diff
@@ -38,7 +70,9 @@ export const CellComponent = memo((props: CellComponentProps) => {
             {/**
              * TODO: Adopt custom column width
              */}
-            <DataTable.Cell width={width}>{cell}</DataTable.Cell>
+            <DataTable.Cell width={width} columnIndex={columnIndex} row={row} column={column}>
```

Comment:
we have this wrapped in cellContext. 
I don't think we need to have these values passed down as props